code/modules/hydroponics/grown/nettle.dm

```diff
@@ -110,11 +111,11 @@
 	if(!..())
 		return
 	if(isliving(M))
-		to_chat(M, "<span class='danger'>You are stunned by the powerful acid of [src]!</span>")
+		to_chat(M, "<span class='danger'>You are stunned by the powerful sting of [src]!</span>")
 		log_combat(user, M, "attacked", src)
 
 		M.adjust_blurriness(force/7)
-		if(prob(20))
+		if(prob(10))
```

Comment:
you may or may not know this but the deathnettle's attack effects don't actually trigger right now because they are bugged - they only happen when they're eaten thanks to ordering of attack and magic of OOP (so this change doesn't actually do anything)